app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(peoples: list) -> list:
+    persons = []
+    for person in peoples:
+        persons.append(Person(person['name'], person['age']))
+
+    for people in peoples:
```

Comment:
maybe better to use something like "for person in people", because "people" here is perceived like something plural